src/db/migration/1610566573713-MigrateTables.ts

```diff
@@ -91,7 +105,7 @@ export class MigrateTables1610566573713 implements MigrationInterface {
         );
     }
 
-    public async down(queryRunner: QueryRunner): Promise<void> {
+    public async down(): Promise<void> {
```

Comment:
This might make the `typeorm` gods displeased: https://github.com/typeorm/typeorm/blob/master/src/migration/MigrationInterface.ts

Reply:
Nah, it's fine: https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters-in-callbacks

"In JavaScript, if you call a function with more arguments than there are parameters, the extra arguments are simply ignored. TypeScript behaves the same way."

Reply:
Lol nothing matters

Ok, cool. Thanks for the explanation.